.travis.yml

```diff
@@ -36,6 +36,16 @@ script:
   - export cleanup=false
   # run ci tests
   - ${PWD}/tests/runtests.sh
+  - pip install --upgrade pip
```

Comment:
remove this line

Reply:
I already did that before and the build is still failed.
Maybe I'll try it later.

Reply:
hmm i see this travis using docker. Maybe @hispanico can help you.